ivy/array/experimental/elementwise.py

```diff
@@ -890,6 +890,54 @@ def diff(
             self._data, n=n, axis=axis, prepend=prepend, append=append, out=out
         )
 
+    def ediff1d(
```

Comment:
why is this still here?

Reply:
Since I remove the backend functions. I'm not sure if this function is used in the frontend's function. And I have removed it.